js/jquery.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+
+  function() {
+    $("ul").slideDown("fast");
+  }
+
+
+  $(".menuIcon").click(showNav)
+
+
+
+});
```

Comment:
Keep up the great work! This is really coming together. We're getting to the nit-picky little problems and that's a great place to be. It means you've got the fundamentals down. Keep up the good work and when you're ready click the big green "Merge" button at the bottom of the pull request.